tests/performance_tests/read_log_data.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+ 
+import argparse
+import log_reader
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument("log_path", type=str, help="Path to nodeos log to scrape")
+parser.add_argument("--start_block", type=int, help="First significant block number in the log", default=2)
```

Comment:
Since our common use pattern in `performance_test_basic.py` has us specifying `startBlock` and `ceaseBlock` it might be good to go ahead an mirror that into this script by providing the `--cease_block` as well.